BioArchLinux/snpeff/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: Bipin Kumar <bipin@ccmb.res.in>
+
+pkgname=snpeff
+pkgver=5.1
+pkgrel=1
+pkgdesc="Genetic variant annotation and effect prediction toolbox. https://doi.org/10.4161/fly.19695"
+arch=('any')
+url="https://pcingola.github.io/SnpEff/"
+provides=(""${pkgname%-pkgver}"")
```

Comment:
A pkg named `snpeff` of course provides `snpeff`, aka itself. So this line is not needed.